labels.md

```diff
@@ -0,0 +1,220 @@
+                                            component/bench                     #d1fad7
+C: TiKV-Client                  #d1fad7     component/TiKV client               #d1fad7
+C: TiKV-Ctl                     #d1fad7     component/TiKV ctl                  #d1fad7
+C: Titan                        #d1fad7     component/Titan                     #d1fad7
+C: Txn                          #d1fad7     component/transactions              #d1fad7
+C: Util                         #d1fad7     component/util                      #d1fad7
+C: gRPC                         #d1fad7     component/gRPC                      #d1fad7
+D: Easy                         #0e8a16     difficulty/easy                     #0e8a16
+D: Medium                       #f4b169     difficulty/medium                   #0e8a16
+D: Mentor                       #31c639     difficulty/mentor                   #0e8a16
+P: Critical                     #ed0000     priority/high                       #eb6420
+P: High                         #ed8888     priority/medium                     #eb6420
+P: Low                          #eeee00     priority/low                        #eb6420
+P: Release-blocker              #f25c8e     priority/blocker                    #eb6420
+S: BotClose                     #c6054c     status/bot close                    #e6e6e6
+S: CanMerge                     #4be524     status/bot merge                    #e6e6e6
+S: DNM                          #DDDDDD     status/DNM                          #e6e6e6
+S: Discussion                   #fbca04     type/discussion                     #1d76db
+S: Duplicate                    #dddddd     status/closed dup                   #e6e6e6
+S: HelpWanted                   #fbca04     status/help wanted                  #e6e6e6
+S: Invalid                      #dddddd     status/closed invalid               #e6e6e6
+S: LGT1                         #66d7ee     status/waiting on review            #e6e6e6
+S: LGT2                         #66d7ee     * <status/waiting on review>            
+S: Proposal                     #fbca04     type/proposal                       #1d76db
+S: WIP                          #DDDDDD     status/WIP                          #e6e6e6
+S: Waiting                      #DDDDDD     status/waiting on author            #e6e6e6
+T: Bug                          #d93f0b     type/bug                            #1d76db
+T: BugFix                       #1d76db     type/bug fix                        #1d76db
+T: CHANGELOG                    #006b75     type/CHANGELOG                      #1d76db
+T: CherryPick                   #1d76db     * <type/cherry pick 2.1 or type/cherry pick 3.0>
+T: Contributor ⭐️               #1d76db     type/contributor ⭐️                 #1d76db
+T: Enhancement                  #1d76db     type/enhancement                    #1d76db
+T: NeedCherryPick-2.1           #333333     type/cherry pick 2.1                #1d76db
```

Comment:
`T: NeedCherryPick` is not the same as cheery pick. NeedCherryPick is a hint for bot which means bot *will* create a PR for it later, while `T: CherryPick` means this PR is a cheery pick.